pyirf/tests/test_integration.py

```diff
@@ -0,0 +1,50 @@
+import glob
+import subprocess
+import logging
+import os
+from astropy.io import fits
+from ogadf_schema.irfs import AEFF_2D, EDISP_2D, PSF_TABLE, BKG_2D, RAD_MAX
+
+
+def test_integration_test(caplog):
+
+    caplog.set_level(logging.INFO)
```

Comment:
you can be specific about the logger to use. It should be `fits_schema` here.
Also, a minimum level of `WARNING` seems more appropriate. `fits_schema` uses `logging.WARNING` for unknown stuff, e.g. unexpected header keywords and `logging.ERROR` for violations of the standard.